tools/kompos/src/debugger.ts

```diff
@@ -2,10 +2,14 @@ import {IdMap, Source, SourceCoordinate, SourceMessage, TaggedSourceCoordinate,
   getSectionId} from './messages';
 import {Breakpoint} from './breakpoints';
 
-export function isRelevant(sc: TaggedSourceCoordinate) {
+export function isRelevantSendTag(sc: TaggedSourceCoordinate) {
```

Comment:
this is not proper english. Now the name needs to be started with `has` instead of `is`.